test/functional/apps/visualize/_tag_cloud.ts

```diff
@@ -146,7 +146,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.settings.clickIndexPatternLogstash();
         await PageObjects.settings.filterField(termsField);
         await PageObjects.settings.openControlsByName(termsField);
-        await (await testSubjects.findAll('formRowToggle'))[1].click();
+        await (
```

Comment:
@sebelga Curious if you have a better way of selecting a data-test-subj

Reply:
Not that I know of. In the component integration tests, we can do `formatRow.toggle` but not sure if that is available in the functional tests.